.github/workflows/ci-build.yaml

```diff
@@ -61,35 +61,11 @@ jobs:
       env:
         JAX_NUM_GENERATED_CASES: ${{ matrix.num_generated_cases }}
       run: |
-        [ ${{ matrix.enable-x64 }} = enable-x64 ] && JAX_ENABLE_X64=1 || JAX_ENABLE_X64=0 
+        [ ${{ matrix.enable-x64 }} = enable-x64 ] && JAX_ENABLE_X64=1 || JAX_ENABLE_X64=0
         pip install -e .
         echo "JAX_NUM_GENERATED_CASES=$JAX_NUM_GENERATED_CASES"
         echo "JAX_ENABLE_X64=$JAX_ENABLE_X64"
         if [ $JAX_ENABLE_X64 = 0 ]; then
           pytest -n auto jax/experimental/jax2tf/tests
         fi
         pytest -n auto tests examples
-
-
-  documentation:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.7]
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        sudo apt install -y pandoc
-        python -m pip install --upgrade pip
-        pip install -r docs/requirements.txt
-    - name: Build and test documentation
-      run: |
-        sphinx-build -b html -D nbsphinx_execute=always docs docs/build/html
-        pytest docs
```

Comment:
Weren't there tests here beyond just building the docs? Don't they need to run still?

Reply:
You're right, it's apparently running `doctest` somehow. There might be some way to get RTD to do this for us, but I can't figure it out, so I'll revert this change for now.

Reply:
https://github.com/google/jax/pull/3497